packages/ui/src/modal/buy/BuyModal.tsx

```diff
@@ -65,7 +70,8 @@ const ModalCopy = {
 
 type Props = Pick<Parameters<typeof Modal>['0'], 'trigger'> & {
   openState?: [boolean, Dispatch<SetStateAction<boolean>>]
-  creditCardCheckoutButton?: JSX.Element
+  creditCardCheckoutComponent?: JSX.Element
+  enableCreditCardCheckout?: boolean
```

Comment:
We can probably omit this, if they want the cc checkout component we should assume that they want it enabled.